src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/Example3.java

```diff
@@ -0,0 +1,22 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MatchXpath">
+      <property name="query"
+       value="//METHOD_DEF[./IDENT[@text='test' or @text='foo']]"/>
+      <message key="matchxpath.match"
+      value="Method name should not be 'test' or 'foo'"/>
```

Comment:
Move values more to the right

Reply:
should I also move the value for the message key  to the right by 4 space?

Reply:
Yes, such indentation help a lot when people read code. Check website to see that config became a bit messy

Reply:
Done.